charts/gardener/provider-local/internal/machine-controller-manager/seed/templates/deployment.yaml

```diff
@@ -121,7 +121,6 @@ spec:
             cpu: 50m
             memory: 64Mi
           limits:
-            cpu: 350m
```

Comment:
We see that a cpu limit is not removed from `./pkg/provider-local/controller/network/actuator.go`. Is it because the corresponding container is with guaranteed QoS? And, a question in general, do we also remove the CPU limits from guaranteed QoS Pods? If not, I guess it would makes sense to document this handlings in a developer guide.

Reply:
The idea was to not remove CPU limits from guaranteed QoS pods. Did you notice such changes? What exactly do you want documented in the developer guide, and why?

Reply:
> Did you notice such changes?

I didn't but looks like you did - https://github.com/gardener/gardener/compare/8226c5dff21f937c53ab9a7972e6b2e221e111aa..f9062639169187226bc4d4d3a75213c861c34bff. Thanks!

> From my perspective, it is self-evident that a pod with a guaranteed QoS should have limits (since guaranteed means requests == limits).

Hm, it was not self-evident for me and @plkokanov . But it can be the case just because we miss context on the discussions. IMO it would make sense to put somewhere few sentences for developers/maintainers to do not add a CPU limits for a Burstable Pod (with the appropriate reasoning). Otherwise we risk the rule imposed by this PR to be violated for a new component that gets introduced tomorrow.

Reply:
This PR is not intended to impose any rule. In the discussion that we had the consensus is as follows. There is nothing wrong with having limits in general. If a component owner knows a good limit (e.g. because he knows the code), it's perfectly ok to have one. Our current issue with limits is that they are mostly based on nothing, and therefore hurt much more than help. 

Therefore, I will not document that there should be no CPU limits, because I myself don't think this is the case, and neither do the participants in the discussion.